### PR TITLE
update - mobile section complete

### PR DESCRIPTION
**Pull request by Salwa99**
## Summary
- completed the mobile section
- added three new sections
    - Portfolio
    - Languages
    - Contact
- used two major frameworks
    - Flex
    - Grid

## Changelog
- `mobile-section` ---> `main`

### CODE REVIEW

**Comment by Salwa99** [comment]
finishing the first section of the Portfolio's mobile version

File: index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    </style>
+</head>
+<body>
+    <header>
+        <nav>
+            <input type="checkbox" id="check">
+            <label for="check" class="checkbtn">
+                <i class="fas fa-bars"></i>
+            </label>
+            <label class="logo">My logo</label>
```
Comment (Qoosim) [inline index.html:25]: - [ ] Every logo on a website has a link to the home page. That's the standard. Kindly change the `<label>` to `<a>` tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        }
+    </style>
+</head>
+<body>
+    <header>
+        <nav>
+            <input type="checkbox" id="check">
+            <label for="check" class="checkbtn">
+                <i class="fas fa-bars"></i>
+            </label>
```
Comment (Qoosim) [inline index.html:24]: - [ ] It is required to wrap your hamburger inside `<nav>` . Kindly change the `<label>` tag to `<nav>` tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        </p>
+        <p class="connect">Let's connect</p>
+        <div class="social-icon">
+                <a ref="#"><img src="style/img/twitter.png" class="icon-image" alt="twitter icon"/></a>
+                <a ref="#"><img src="style/img/m.png" class="icon-image" alt="m icon"/></a>
+                <a ref="#"><img src="style/img/github.png" class="icon-image" alt="github icon"/></a>
+                <a href="#"><img src="style/img/linkedin.png" class="icon-image"></img></a>
+                <a href="#"><img src="style/img/v.png" class="icon-image" alt="v icon"/></a>
+                
+        </div>
```
Comment (Qoosim) [inline index.html:49]: - [ ] It is required that the list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag). Kindly implement this suggestion to this part of the portfolio.

**Review by Qoosim — COMMENTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @Salwa99, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-  Good  README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] For some reason, your linter files do not run on your PR, I can see the set up though. Kindly look into what the issue is and fix.

- [ ] It is require to produce a close replica of the chosen template. This includes  font-size, font-family, font-weight, background color, color, padding, margin among others. There are few things you still need to look into on the Figma template and work your design along that line. Below are the screenshots of both your design and the Figma template.

### Your Design

your-design


### Figma Design

figma-temp

File: index.html (hunk 3 above)
Comment (Salwa99) [reply index.html:49]: thank you @Qoosim  for the accurate description, I will fix the missing parts

**Pull request merged**